js/list-view.js

```diff
@@ -1,86 +1,58 @@
-define([
-  'core/js/adapt',
-  'core/js/views/componentView'
-], function (Adapt, ComponentView) {
+import ComponentView from 'core/js/views/componentView';
 
-  var ListView = ComponentView.extend({
+class ListView extends ComponentView {
 
-    preRender: function () {
-      this.checkIfResetOnRevisit();
-      this.listenTo(Adapt, 'device:resize', this.resizeControl);
-    },
+  className() {
+    let classes = super.className();
 
-    postRender: function () {
-      this.setReadyStatus();
-
-      this.setupInviewCompletion('.component__widget');
-
-      this.setUpColumns();
-
-      if (!this.model.get('_animateList')) return;
-
-      this.$el.addClass('is-animated-list');
-      this.$('.list__container').on('onscreen.animate', this.checkIfOnScreen.bind(this));
-    },
-
-    resizeControl: function () {
-      this.setUpColumns();
-    },
-
-    setUpColumns: function () {
-      var columns = this.model.get('_columns');
-
-      if (columns < 2) return;
-
-      var isLarge = Adapt.device.screenSize === 'large';
-      this.$('.list__item').css('width', isLarge ? (100 / columns) + '%' : '');
-    },
+    if (this.model.get('_animateList')) {
+      classes += ' is-animated-list';
+    }
 
-    checkIfResetOnRevisit: function () {
-      var isResetOnRevisit = this.model.get('_isResetOnRevisit');
+    return classes;
+  }
 
-      // If reset is enabled set defaults
-      if (isResetOnRevisit) {
-        this.model.reset(isResetOnRevisit);
-      }
-    },
+  preRender() {
+    this.listenTo(this.model.getChildren(), { all: this.changed });
 
-    /**
-     * Kicks off the list item animation once the list container is at least xx% on screen
-     */
-    checkIfOnScreen: function (event, measurements) {
-      const percentage = this.model.get('_percentInviewVertical') || 70;
-      if (measurements.percentInviewVertical < percentage) return;
+    super.preRender();
+  }
 
```

Comment:
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/adaptView.js#L26

You can delete this function